spec/features/notes_spec.rb

```diff
@@ -23,12 +23,12 @@
       visit project_path(project)
 
       within('#in_progress .story') do
-        find('.story-title').trigger('click')
+        find('.story-title').click
         fill_in 'note', with: 'Adding a new note'
         click_on 'Add note'
       end
 
-      sleep 0.5
+      wait_for_ajax
```

Comment:
We can have something similar to the `wait_spinner` here for the edit state that disables the fields while saving. We have it for v1 as well, right @kostadriano?